agent/task.go

```diff
@@ -228,8 +229,23 @@ func (a *Agent) getTaskConfig(ctx context.Context, tc *taskContext) (*model.Task
 		return nil, errors.New("agent retrieved an empty project ref")
 	}
 
+	var confPatch *patch.Patch
+	if confVersion.Requester == evergreen.GithubPRRequester {
+		tc.logger.Execution().Info("Fetching patch.")
```

Comment:
"Fetching patch for GitHub pull request"

Reply:
done